acceptance/tests/resource/ssh_authorized_key/modify.rb

```diff
@@ -30,6 +30,7 @@
   step "verify entry updated in #{auth_keys}"
   on(agent, "cat #{auth_keys}")  do |res|
     fail_test "didn't find the updated key for #{name}" unless stdout.include? "mynewshinykey #{name}"
+    fail_test "Found old key mykey #{name}" if stdout.include? "mykey #{name}"
```

Comment:
This is necessary because in a case where the original key didn't exist, Puppet will just make a new 'myshinynewkey' and fool this test into passing. We want to make sure the old 'mykey' doesn't exist anymore. This came up due to the key being weirdly appended to the end of the previous key.